appmanifest/shortcuts-member/shortcuts-member-skip-for-out-of-scope-url.webmanifest

```
@@ -0,0 +1,17 @@
+{
+    "shortcuts": [{
```

I'd suggest adding an explicit scope everywhere. 

```suggestion
    "scope": "/",
    "shortcuts": [{
```